sharding/contracts/sharding_manager.sol

```diff
@@ -51,6 +51,9 @@ contract SMC {
     uint sampleSizeLastUpdatedPeriod;
 
     // Constant values
+    // Length of challenge period for notary's proof of custody
```

Comment:
Why does it not do the same for period length or shard_count? is it because it is public?

Reply:
Yeah. Only `SHARD_COUNT` is public. We won't be able to see `PERIOD_LENGTH` in `sharding_manager.go` since it's not public